### DISCUSSION TITLE
HPP-FCL 2.3.4

### DISCUSSION THREAD

**Discussion by jcarpent** [coal-library/coal#433] [Announcements]
## What's Changed
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/humanoid-path-planner/hpp-fcl/pull/414
* Fix conversion warning by @wxmerkt in https://github.com/humanoid-path-planner/hpp-fcl/pull/417
* Add missing boost include by @nim65s in https://github.com/humanoid-path-planner/hpp-fcl/pull/418
* ci: update macos-linux-pip by @nim65s in https://github.com/humanoid-path-planner/hpp-fcl/pull/419
* Modernize Cmake use by @nim65s in https://github.com/humanoid-path-planner/hpp-fcl/pull/420
* tests: use boost::filesystem by @nim65s in https://github.com/humanoid-path-planner/hpp-fcl/pull/424
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/humanoid-path-planner/hpp-fcl/pull/425
* Update minimal Python version  by @jcarpent in https://github.com/humanoid-path-planner/hpp-fcl/pull/427
* Sync submodule cmake by @jcarpent in https://github.com/humanoid-path-planner/hpp-fcl/pull/430
* Sync submodule CMake by @jcarpent in https://github.com/humanoid-path-planner/hpp-fcl/pull/431

**Full Changelog**: https://github.com/humanoid-path-planner/hpp-fcl/compare/v2.3.3...v2.3.4

<hr /><em>This discussion was created from the release <a href='https://github.com/humanoid-path-planner/hpp-fcl/releases/tag/v2.3.4'>HPP-FCL 2.3.4</a>.</em>

**Comment by nim65s** [comment]
Trying to push this to AUR, I get:
```python
Traceback (most recent call last):
  File "…/hpp-fcl-2.3.4/doc/python/doxygen_xml_parser.py", line 889, in <module>
    index.parseCompound()
  File "…/hpp-fcl-2.3.4/doc/python/doxygen_xml_parser.py", line 693, in parseCompound
    obj = ClassCompound(compound, self)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "…/hpp-fcl-2.3.4/doc/python/doxygen_xml_parser.py", line 436, in __init__
    name=self._className() + "::" + memberdef.find("name").text,
         ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
TypeError: can only concatenate str (not "NoneType") to str
```

I'll investigate a bit further before opening issue / PR here, or fix directly on Arch.

**Reply by nim65s** [reply]
After a workaround for this, I get:
```
…/build-2.3.4/python/doxygen_autodoc/hpp/fcl/narrowphase/gjk.h:60:38: erreur: « const char* doxygen::member_func_doc » redéclaré comme une sorte différente d'entité
   60 | inline const char* member_func_doc ( Status (hpp::fcl::details::EPA::*function_ptr) ( hpp::fcl::details::GJK &, const hpp::fcl::Vec3f &))
      |                                      ^~~~~~
…/build-2.3.4/python/doxygen_autodoc/hpp/fcl/narrowphase/gjk.h:46:20: note: déclaration précédente de « const char* doxygen::member_func_doc(void (hpp::fcl::details::EPA::*)()) »
   46 | inline const char* member_func_doc (void (hpp::fcl::details::EPA::*function_ptr) ())
      |                    ^~~~~~~~~~~~~~~
…/build-2.3.4/python/doxygen_autodoc/hpp/fcl/narrowphase/gjk.h:60:70: erreur: expected unqualified-id before « * » token
   60 | inline const char* member_func_doc ( Status (hpp::fcl::details::EPA::*function_ptr) ( hpp::fcl::details::GJK &, const hpp::fcl::Vec3f &))
      |                                                                      ^
…/build-2.3.4/python/doxygen_autodoc/hpp/fcl/narrowphase/gjk.h:60:71: erreur: « function_ptr » n'a pas été déclaré dans cette portée
   60 | inline const char* member_func_doc ( Status (hpp::fcl::details::EPA::*function_ptr) ( hpp::fcl::details::GJK &, const hpp::fcl::Vec3f &))
      |                                                                       ^~~~~~~~~~~~
…/build-2.3.4/python/doxygen_autodoc/hpp/fcl/narrowphase/gjk.h:60:38: erreur: « Status » n'a pas été déclaré dans cette portée; vouliez-vous employer « PyStatus » ?
   60 | inline const char* member_func_doc ( Status (hpp::fcl::details::EPA::*function_ptr) ( hpp::fcl::details::GJK &, const hpp::fcl::Vec3f &))
      |                                      ^~~~~~
      |                                      PyStatus
  ```
  
While everything seems to work on robotpkg & nix. I think I'll just deactivate doxygen on AUR for now.
